cfgov/templates/welcome.html

```diff
@@ -0,0 +1,17 @@
+{% overextends "wagtailadmin/login.html" %}
+{% load i18n admin_static %}
+
+{% block furniture %}
+    <div class="content-wrapper">
+        <h1>{% trans 'Welcome' %} <br/>
+        You've logged in to the consumerfinance.gov editorial tools.	
+        </h1>
+        <h2>Continue on to...</h2>
+        <ul>
+        {% for name, url in destinations %}
+            <li><a href="{{url}}"> {{name}}</a></li>
+        {% endfor %}
+        </ul>
+        <p>{% trans 'Otherwise, you may want to' %} <a href="/logout/">{% trans 'LOG OUT ' %}</a></p>
```

Comment:
Minor thing, but the wording of "Otherwise" doesn't quite make sense anymore since before it was if you had insufficient permissions.